src/z3c/testing/app.py

```diff
@@ -11,10 +11,9 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""
-$Id$
-"""
-
+from zope.container.tests.test_icontainer import BaseTestIContainer as BTIC
+from zope.container.tests.test_icontainer import DefaultTestData
+from zope.interface.verify import verifyObject, verifyClass
 import unittest
```

Comment:
PEP-8 requires `import unittest` to be above 3rd-party library imports, separated by one blank line.